src/diffpy/labpdfproc/labpdfprocapp.py

```diff
@@ -21,12 +21,13 @@ def _define_arguments():
                 "The filename(s) or folder(s) of the datafile(s) to load. "
                 "Required.\n"
                 "Supply a space-separated list of files or directories. "
+                "If a filename contains whitespace, enclose it in quotes. "
```

Comment:
maybe mention that, as a general matter, avoid filenames with whietspaces in, but if it does, then enclose in quotes